### PR TITLE
Roumec/display score

### PR DESCRIPTION
**Pull request by sebsheep**

### CODE REVIEW

File: webapp/app.py [hunk 1]
```diff
@@ -58,7 +58,8 @@ def home():
                 chapter,
                 user,
             )
-    return flask.render_template('home.html', chapters=chapters)
+        score, max = answers_module.compute_score(chapter,user)
```
Comment (sebsheep) [inline webapp/app.py]: @broumec this is not valid! You only take into account the last score/max!

You can either dynamically add a field `score` to each `chapter` object (it is a bit dirty) or create a dictionary/list containing all the data about the scores.

**Review by sebsheep — COMMENTED** [review]
You have to deal correctly with the score!

File: webapp/app.py (hunk 1 above)
Comment (broumec) [reply webapp/app.py]: I changed it, can you see my modification ?

Comment (sebsheep) [reply webapp/app.py]: Yes, seems great!

**Pull request merged**